src/components/Dropdown/__stories__/dropdown.stories.mdx

```diff
@@ -550,6 +550,110 @@ Inside the advanced filters, a user can select multiple people from the dropdown
   </Story>
 </Canvas>
 
+// TODO demo
```

Comment:
Should all the todos be removed?

Reply:
Will be removed - used them just for demo